docs/content/ecs/logging.md

```diff
@@ -58,13 +58,15 @@ public class MyCustomDispatcher: ILogDispatcher
 ```
 
 
-If you want a specific worker to use the dispatcher, pass it to the constructor of the `WorkerBase` class instead of the default dispatcher:
+If you want a specific worker to use the dispatcher, pass it to the `Connect` method of the `WorkerConnectorBase` class instead of the default dispatcher when connecting to SpatialOS:
 
 ```csharp
-public class UnityClient : WorkerBase
+public class ClientWorkerConnector : WorkerConnectorBase
 {
-    public UnityClient(string workerId, Vector3 origin) : base(workerId, origin, new MyCustomDispatcher())
+    private async void Start()
     {
+        ...
```

Comment:
Nit; May wrap change `...` to `/* ... */` so that it's technically still valid code. ;)